sdk/monitor/azure-monitor-opentelemetry-exporter/src/main/java/com/azure/monitor/opentelemetry/exporter/implementation/quickpulse/model/OpenTelDataPoint.java

```diff
@@ -0,0 +1,17 @@
+package com.azure.monitor.opentelemetry.exporter.implementation.quickpulse.model;
+
+public class OpenTelDataPoint {
```

Comment:
See https://github.com/Azure/azure-sdk-for-java/pull/41220/files#r1688754048. Not sure this class is needed.

Reply:
Each data point that is collected also comes with their associated dimensions, which need to also be stored. I am not storing them right now but will be doing so in the OTelDataPoint class once I move on to implementing filtering on dimensions.